sdk/template-LLC/Azure.Template.LLC/src/swagger/swagger.json

```diff
@@ -1,123 +1,121 @@
             "schema": {
-              "type": "string"
+              "$ref": "#/definitions/Resource"
             }
           }
         }
       },
-      "put": {
-        "operationId": "RequestBodyResponseBody",
-        "description": "RequestBody and ResponseBody",
+      "delete": {
+        "operationId": "Delete",
+        "description": "Delete resource.",
         "parameters": [
           {
-            "name": "model",
-            "description": "body",
-            "in": "body",
-            "schema": {
-              "$ref": "#/definitions/TestModel"
-            }
+            "name": "resourceId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The id of the resource."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          }
+        }
+      },
+      "get": {
+        "operationId": "Get",
+        "description": "Retrieves information about the resource.",
+        "parameters": [
+          {
+            "name": "resourceId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The id of the resource."
           }
         ],
         "responses": {
           "200": {
-            "description": "String Response",
+            "description": "OK",
             "schema": {
-              "$ref": "#/definitions/TestModel"
+              "$ref": "#/definitions/Resource"
             }
           }
         }
       }
     },
-    "/test2": {
+    "/providers/Microsoft.Template/resource": {
       "get": {
-        "operationId": "NoRequestBodyNoResponseBody",
-        "description": "No RequestBody and No ResponseBody",
+        "operationId": "ListResources",
+        "description": "Retrieves the list of resources.",
         "responses": {
           "200": {
-            "description": "Empty Response"
-          }
-        }
-      },
-      "put": {
-        "operationId": "RequestBodyNoResponseBody",
-        "description": "RequestBody and No ResponseBody",
-        "parameters": [
-          {
-            "name": "stringBody",
-            "description": "string body",
-            "in": "body",
+            "description": "OK",
             "schema": {
-              "type": "string",
-              "x-nullable": true
+              "$ref": "#/definitions/ResourceListResult"
             }
           }
-        ],
-        "responses": {
-          "200": {
-            "description": "Empty Response"
-          }
         }
       }
     }
   },
   "definitions": {
-    "TestModel": {
-      "type": "object",
+    "Resource": {
       "properties": {
-        "Code": { "type": "string" },
-        "Status": { "type": "string" }
+        "name": {
+          "type": "string"
+        },
+        "id": {
+          "type": "string"
+        }
       },
       "description": "."
+    },
+    "ResourceListResult": {
```

Comment:
This type should have a nextLink in it... see Example 1 under https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-pageable